### DISCUSSION TITLE
Bitbucket deployment intergration

### DISCUSSION THREAD

**Discussion by Noaber** [andreapollastri/cipi#366] [Ideas]
First of all, awesome project! Thanks!

As for the future, do you plan to have Bitbucket intergration / deployment just like Ploi.io ?
Reason to ask is that I have all my code on bitbucket (this started before Git has private repo's) and it would be awesome to deploy from Bitbucket instead of Git.

Regards,

**Comment by andreapollastri** [comment]
Hi, Thank You for the idea! I'll evaluate it but currently there is no plan to support bitbucket or Gitlab. I'm sorry.. Maybe in v4 but not in 3! I'll update you!

**Reply by Noaber** [reply]
Thanks for your answer and consideration eventually for v4. When the time is there, I will hear from you ;)

**Comment by andreapollastri** [comment]
Hello guys, it the last 3 years the world has been changed with AI and with new deployments needs so a new "Laravel Control Panel" is not the real solution for today's world. I have created a new stable version of Cipi full of features and based on a powerful and light CLI. If you would like to try it, here you can find all informations: https://cipi.sh/
